### PR TITLE
[flutter_splash_tizen] Initial implementation

### PR DESCRIPTION
**Pull request by JRazek**
Issue #151 
Adds support for native splash as a flutter package.

### CODE REVIEW

File: packages/flutter_splash_tizen/example/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  flutter_test:
+    sdk: flutter
+
+  flutter_lints: ^1.0.0
+
+flutter:
+  uses-material-design: true
+
+flutter_splash_tizen:
+  image: assets/test.png
```
Comment (pkosko) [inline packages/flutter_splash_tizen/example/pubspec.yaml:28]: no new line at the end of file

File: packages/flutter_splash_tizen/example/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    initPlatformState();
+  }
+
+  // Platform messages are asynchronous, so we initialize in an async method.
+  Future<void> initPlatformState() async {
+    String platformVersion;
+    // Platform messages may fail, so we use a try/catch PlatformException.
+    // We also handle the message potentially returning null.
+    try {
+      platformVersion =
```
Comment (pkosko) [inline packages/flutter_splash_tizen/example/lib/main.dart:33]: is the platformVersion really a part of splash plugin? or is it just leftover after some base for development?

File: packages/flutter_splash_tizen/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  flutter:
+    sdk: flutter
+  image: ^3.0.8
+  xml: ^5.3.1
+  yaml: ^3.1.0
+    
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.0
```
Comment (pkosko) [inline packages/flutter_splash_tizen/pubspec.yaml:20]: new line missing at the end of file

File: packages/flutter_splash_tizen/test/flutter_splash_tizen_test.dart [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    channel.setMockMethodCallHandler((MethodCall methodCall) async {
+      return '42';
+    });
+  });
+
+  tearDown(() {
+    channel.setMockMethodCallHandler(null);
+  });
+
+  test('getPlatformVersion', () async {
```
Comment (pkosko) [inline packages/flutter_splash_tizen/test/flutter_splash_tizen_test.dart]: are these tests related to splash screen?

File: packages/flutter_splash_tizen/lib/flutter_splash_tizen.dart [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+// specifying the `--platforms` flag. A plugin project supports no platforms is generated.
+// To add platforms, run `flutter create -t plugin --platforms <platforms> .` under the same
+// directory. You can also find a detailed instruction on how to add platforms in the `pubspec.yaml` at https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class FlutterNativeSplashTizen {
+  static const MethodChannel _channel =
```
Comment (pkosko) [inline packages/flutter_splash_tizen/lib/flutter_splash_tizen.dart]: is it related to splash screen?

File: packages/flutter_splash_tizen/bin/create.dart [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import 'file_utils.dart';
+import 'package:xml/xml.dart';
+
+void main() {
+  var doc = loadYamlFileSync("pubspec.yaml")?['flutter_splash_tizen'];
```
Comment (pkosko) [inline packages/flutter_splash_tizen/bin/create.dart:5]: if I understand correctly, current flow supports only adding a file once, then file added to yaml will be copied to tizen-manifest. What about changing a file mentioned in yaml during the development? Shouldn't we consider a scenario that a file is added to manifest file everytime when application is built which means that the file used in tizen manifest will be kept in sync with file mentioned in pubspec.yaml file?

**Review by pkosko — CHANGES_REQUESTED** [review]

File: packages/flutter_splash_tizen/example/pubspec.yaml (hunk 1 above)
Comment (pkosko) [reply packages/flutter_splash_tizen/example/pubspec.yaml:28]: one more comment - I assume that the example application is after adding a splash file with flutter-tizen pub run flutter_splash_tizen:create because related line is added to tizen-manifest.xml. But file assets/test.png does not exist, so removing splash and re-adding it again will probably fail - please check it.

I am not also sure if packages/flutter_splash_tizen/example/tizen/shared/res/test.png file will work as a splash screen, as the tizen-manifest file uses: assets/test.png.

**Review by pkosko — CHANGES_REQUESTED** [review]

**Comment by swift-kim** [comment]
By the way this package is not a plugin. I need to make a decision on which repository this package should be placed in.

**Comment by swift-kim** [comment]
I discussed with @WonyoungChoi and we decided to keep this package in this repo for now. The decision may be changed in the future.

File: packages/flutter_splash_tizen/bin/create.dart (hunk 6 above)
Comment (pkosko) [reply packages/flutter_splash_tizen/bin/create.dart:5]: @swift-kim could you please share your opinion regarding this comment? Should we set splash screen 'on demand' only, or rather do it automatically on every build?

File: packages/flutter_splash_tizen/bin/create.dart [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+  splashScreens.children.clear();
+  XmlElement splashScreen = XmlElement(XmlName("splash-screen"));
+  splashScreen.setAttribute("src", image);
+  splashScreen.setAttribute("type", "img");
+  splashScreen.setAttribute("indicator-display", "false");
+  splashScreen.setAttribute("orientation", "portrait");
+  splashScreens.children.add(splashScreen);
+  
+  writeToFileSync(tizenManifestPath, el.toXmlString());
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/create.dart:32]: How about using `writeAsStringSync` of  `dart:io`? You don't need to check the file because this method will create new one if the file does not exist.
```suggestion
  File(tizenManifestPath).writeAsStringSync(el.toXmlString());
```

File: packages/flutter_splash_tizen/bin/file_utils.dart [hunk 8]
```diff
@@ -0,0 +1,28 @@
+import 'dart:io';
+import 'package:yaml/yaml.dart';
+import 'package:xml/xml.dart';
+
+Map? loadYamlFileSync(String path) {
+  File file = File(path);
+  if (file.existsSync() == true) {
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/file_utils.dart:7]:
```suggestion
  if (file.existsSync()) {
```

File: packages/flutter_splash_tizen/bin/file_utils.dart [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+    return loadYaml(file.readAsStringSync());
+  }
+  return null;
+}
+
+XmlDocument loadXMLFileSync(String path) {
+  File file = File(path);
+  if (file.existsSync() == true) {
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/file_utils.dart:15]:
```suggestion
  if (file.existsSync()) {
```

File: packages/flutter_splash_tizen/bin/file_utils.dart [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    return XmlDocument.parse(file.readAsStringSync());
+  }
+  throw FormatException("could not read $path");
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/file_utils.dart:18]: I think returning `null` is better. 
```suggestion
  return null;
```

File: packages/flutter_splash_tizen/bin/remove.dart [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import 'file_utils.dart';
+import 'package:xml/xml.dart';
+
+void main() {
+  var doc = loadYamlFileSync("pubspec.yaml")?['flutter_splash_tizen'];
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/remove.dart:5]: unnecessary loading yaml file.

File: packages/flutter_splash_tizen/bin/create.dart (hunk 6 above)
Comment (WonyoungChoi) [reply packages/flutter_splash_tizen/bin/create.dart:5]: I think it's better to sync automatically every build if possible. By the way, is there a way to run this package every build?
Should we add this feature to `flutter-tizen` tool?

Comment (WonyoungChoi) [reply packages/flutter_splash_tizen/bin/create.dart:5]: `doc` can be `null`. please check it.

**Review by WonyoungChoi — CHANGES_REQUESTED** [review]

File: packages/flutter_splash_tizen/bin/create.dart (hunk 6 above)
Comment (swift-kim) [reply packages/flutter_splash_tizen/bin/create.dart:5]: @pkosko You can just follow flutter_native_splash's way. So manually running the pub run command every time the attributes are changed should just be enough.

File: packages/flutter_splash_tizen/LICENSE [hunk 12]
```diff
@@ -0,0 +1,26 @@
+Copyright 2013 The Flutter Authors. All rights reserved.
+Copyright (c) 2021, Samsung Electronics Co., Ltd. All rights reserved.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/LICENSE:2]: Flutter Authors?

File: packages/flutter_splash_tizen/LICENSE [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above
+      copyright notice, this list of conditions and the following
+      disclaimer in the documentation and/or other materials provided
+      with the distribution.
+    * Neither the name of Google Inc. nor the names of its
+      contributors may be used to endorse or promote products derived
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/LICENSE:14]: Google?

File: packages/flutter_splash_tizen/README.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
+# flutter_splash_tizen
+
+Flutter-tizen package that allows to add custom splash-screen images into your apps.
+## Getting Started
+
+First you should add the package to dependencies section in your `pubspec.yaml`
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/README.md:6]:
```suggestion
First you should add the package to dev_dependencies section in your `pubspec.yaml`.
```

File: packages/flutter_splash_tizen/README.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+```yaml
+dependencies:
+  flutter_splash_tizen: ^0.0.1
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/README.md:10]:
```suggestion
dev_dependencies:
  flutter_splash_tizen: ^0.1.0
```

File: packages/flutter_splash_tizen/README.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+```
+After that run 
+```
+flutter-tizen pub get
+```
+in order to download all the dependencies.<br>
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/README.md:16]:
```suggestion
in order to download the dependency.
```
Please remove all unnecessary `<br>`s in this file.

File: packages/flutter_splash_tizen/analysis_options.yaml [hunk 17]
```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
+
+# Additional information about this file can be found at
+# https://dart.dev/guides/language/analysis-options
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/analysis_options.yaml:4]: Remove the comments.

File: packages/flutter_splash_tizen/bin/file_utils.dart [hunk 18]
```diff
@@ -0,0 +1,28 @@
+import 'dart:io';
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/bin/file_utils.dart:1]: Source files should be placed inside `lib` or `lib/src` but not `bin`.

File: packages/flutter_splash_tizen/example/.metadata [hunk 19]
```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/.metadata:1]: Please remove this file.

File: packages/flutter_splash_tizen/example/README.md [hunk 20]
```diff
@@ -0,0 +1,16 @@
+# flutter_splash_tizen_example
+
+Demonstrates how to use the flutter_splash_tizen plugin.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/README.md:3]: This package is not a plugin.

File: packages/flutter_splash_tizen/example/README.md [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/README.md:16]: Remove

File: packages/flutter_splash_tizen/example/analysis_options.yaml [hunk 22]
```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/analysis_options.yaml:1]: Remove any comments.

File: packages/flutter_splash_tizen/example/pubspec.yaml [hunk 23]
```diff
@@ -0,0 +1,28 @@
+name: flutter_splash_tizen_example
+description: Demonstrates how to use the flutter_splash_tizen plugin.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/pubspec.yaml:2]: Not a plugin.

File: packages/flutter_splash_tizen/example/pubspec.yaml [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  flutter_splash_tizen:
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/pubspec.yaml:13]: Move this out to dev_dependencies.

File: packages/flutter_splash_tizen/example/pubspec.yaml [hunk 25]
```diff
@@ -0,0 +1,28 @@
...
+    path: ../
+
+  cupertino_icons: ^1.0.2
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/pubspec.yaml:16]: Sort the dependencies alphabetically and remove extra newlines between the dependencies.

File: packages/flutter_splash_tizen/pubspec.yaml [hunk 26]
```diff
@@ -0,0 +1,21 @@
+name: flutter_splash_tizen
+description: A new flutter plugin project.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/pubspec.yaml:2]: Insufficient description.

File: packages/flutter_splash_tizen/pubspec.yaml [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+homepage:
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=1.20.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  image: ^3.0.8
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/pubspec.yaml:13]: Unused dependency?

File: packages/flutter_splash_tizen/pubspec.yaml [hunk 28]
```diff
@@ -0,0 +1,21 @@
+name: flutter_splash_tizen
+description: A new flutter plugin project.
+version: 0.0.1
+homepage:
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/pubspec.yaml:4]: Specify homepage and repository as other packages in this repo do.

File: packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml [hunk 29]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.flutter_splash_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="wearable"/>
+    <ui-application appid="com.example.flutter_splash_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true" api-version="4">
+        <label>flutter_splash_tizen_example</label>
+        <icon>ic_launcher.png</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+    </ui-application>
+    <tv-info api-version="8.8.0">
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml:9]: The example app is expected to work out of the box without requiring additional actions from users. Please add the `<splash-screens>` section to tizen-manifest.xml.

File: packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml [hunk 30]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.flutter_splash_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="wearable"/>
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml:3]: Does this package support TV as well?

**Review by swift-kim — CHANGES_REQUESTED** [review]
Issue:
A blank (black) screen is displayed before the first app frame is drawn and after the splash screen is displayed.

Additional work:
- Adding to labeler (https://github.com/flutter-tizen/plugins/blob/master/.github/labeler.yml).
- Adding to recipe (https://github.com/flutter-tizen/plugins/blob/master/tools/recipe.yaml).
- Adding to README (https://github.com/flutter-tizen/plugins/blob/master/README.md).
- Resolving any CI errors.

**Comment by HakkyuKim** [comment]
You can exclude the package from integration test by fixing `.github/workflows/integration_test.yml`. Add this package name like this in both steps: "Run tests for changed plugins" and "Run tests for all plugins":

```diff
./tools/run_command.py test \
  --recipe ./tools/recipe.yaml \
  --generate-emulators \
  --run-on-changed-packages \
  --base-sha=$(git rev-parse HEAD^) \
  --exclude wearable_rotary image_picker camera webview_flutter \
     video_player permission_handler geolocator battery connectivity \
     device_info package_info sensors share wifi_info_flutter \
-     google_maps_flutter tizen_app_control url_launcher network_info_plus
+     google_maps_flutter tizen_app_control url_launcher network_info_plus flutter_splash_tizen
```

Also write a comment why the test is excluded, something like "flutter_splash_tizen: non plugin package.".

**Comment by HakkyuKim** [comment]
@JRazek 
You also need to add it in the "Run tests for changed plugins" step.

File: .github/workflows/integration_test.yml [hunk 31]
```diff
@@ -43,7 +43,7 @@ jobs:
             --exclude wearable_rotary image_picker camera webview_flutter \
               video_player permission_handler geolocator battery connectivity \
               device_info package_info sensors share wifi_info_flutter \
-              google_maps_flutter tizen_app_control url_launcher network_info_plus
+              google_maps_flutter tizen_app_control url_launcher network_info_plus flutter_splash_tizen
```
Comment (swift-kim) [inline .github/workflows/integration_test.yml:46]:
```suggestion
              google_maps_flutter tizen_app_control url_launcher network_info_plus \
              flutter_splash_tizen
```

File: .github/workflows/integration_test.yml [hunk 32]
```diff
@@ -53,3 +53,4 @@ jobs:
         #   battery, connectivity, device_info, package_info, sensors, share, wifi_info_flutter: deprecated.
         #   google_maps_flutter, webview_flutter, url_launcher, network_info_plus: not supported by emulators
         #   tizen_app_control: test available after Flutter 2.5 migration.
+        #   flutter_splash_tizen: non plugin package.
```
Comment (swift-kim) [inline .github/workflows/integration_test.yml:56]: Packages can also have tests. You could move flutter_splash_tizen to line 48.

File: packages/flutter_splash_tizen/LICENSE [hunk 33]
```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2021, Samsung Electronics Co., Ltd. All rights reserved.
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/LICENSE:1]: Please copy this file as it is:

https://github.com/flutter-tizen/plugins/blob/master/packages/tizen_app_control/LICENSE

File: packages/flutter_splash_tizen/analysis_options.yaml [hunk 34]
```diff
@@ -0,0 +1 @@
+include: package:flutter_lints/flutter.yaml
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/analysis_options.yaml:1]: Add newlines at the end of ALL files.

File: packages/flutter_splash_tizen/bin/create.dart [hunk 35]
```diff
@@ -0,0 +1,41 @@
...
+
+  splashScreens.children.clear();
+  XmlElement splashScreen = XmlElement(XmlName("splash-screen"));
+  splashScreen.setAttribute("src", image);
+  splashScreen.setAttribute("type", "img");
+  splashScreen.setAttribute("indicator-display", "false");
+  splashScreen.setAttribute("orientation", "portrait");
+  splashScreens.children.add(splashScreen);
+  
+  File(tizenManifestPath).writeAsStringSync(el.toXmlString());
```
Comment (swift-kim) [inline packages/flutter_splash_tizen/bin/create.dart:40]: Use
```
el.toXmlString(pretty: true, indent: '    ') + '\n'
```

File: .github/workflows/integration_test.yml (hunk 32 above)
Comment (JRazek) [reply .github/workflows/integration_test.yml:56]: @swift-kim What comment should I use then? "Runs manually" would fit?

Comment (HakkyuKim) [reply .github/workflows/integration_test.yml:56]: You could append it to wearable_rotary and image_select.

**Comment by swift-kim** [comment]
Please don't spam commits. You can push multiple commits at once.

File: .github/workflows/integration_test.yml [hunk 36]
```diff
@@ -43,9 +44,10 @@ jobs:
             --exclude wearable_rotary image_picker camera webview_flutter \
               video_player permission_handler geolocator battery connectivity \
               device_info package_info sensors share wifi_info_flutter \
-              google_maps_flutter tizen_app_control url_launcher network_info_plus
+              google_maps_flutter tizen_app_control url_launcher network_info_plus \
+              flutter_splash_tizen
         # The following packages are excluded from tests:
-        #   wearable_rotary, image_picker: no tests.
+        #   wearable_rotary, image_picker: no tests, flutter_splash_tizen: runs manually.
```
Comment (swift-kim) [inline .github/workflows/integration_test.yml:50]: This package doesn't have a test.
```suggestion
        #   wearable_rotary, image_picker, flutter_splash_tizen: no tests.
```

File: packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml (hunk 30 above)
Comment (swift-kim) [reply packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml:3]: Could you confirm whether splash screen is supported on TV devices or not? If not, you should mention the fact in the package's README.

**Comment by HakkyuKim** [comment]
The CI will pass when you rebase to master branch.

File: packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml (hunk 30 above)
Comment (JRazek) [reply packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml:3]: @swift-kim As far as I see - it does not run on TV. It runs on mobile and on Wearable fine.

File: packages/flutter_splash_tizen/bin/remove.dart [hunk 37]
```diff
@@ -0,0 +1,32 @@
...
+  return null;
+}
+
+void main() {
+  String tizenManifestPath = "tizen/tizen-manifest.xml";
+
+  XmlDocument? tizenManifest = loadXMLFileSync(tizenManifestPath);
+  if (tizenManifest == null) {
+    throw const FormatException("could not read tizen-manifext.xml!");
+  }
```
Comment (WonyoungChoi) [inline packages/flutter_splash_tizen/bin/remove.dart:18]: Unnecessary exporting the method `loadXMLFileSync`. 

```suggestion
  File tizenManifestFile = File("tizen/tizen-manifest.xml");
  if (!file.existsSync()) {
    throw const FormatException("could not read tizen-manifext.xml!");
  }
  tizenManifest = XmlDocument.parse(tizenManifestFile.readAsStringSync());
```
```

**Comment by swift-kim** [comment]
Did you read all my comments in the above?

**Comment by JRazek** [comment]
> Did you read all my comments in the above?
Yes,

**Comment by swift-kim** [comment]
Hey. There are more comments.

File: packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml (hunk 30 above)
Comment (JRazek) [reply packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml:3]: Will this work for that?
> Package has been tested on Mobile 6.0, Wearable 6.0 and IoT Headed 6.5 platform.

**Comment by swift-kim** [comment]
Are you serious? Why are you publishing a package which isn't even merged?

https://pub.dev/packages/flutter_splash_tizen

**Comment by swift-kim** [comment]
> Issue:
> A blank (black) screen is displayed before the first app frame is drawn and after the splash screen is displayed.

This issue needs a fix before this package can be merged.

**Comment by bbrto21** [comment]
> > Issue:
> > A blank (black) screen is displayed before the first app frame is drawn and after the splash screen is displayed.
> 
> This issue needs a fix before this package can be merged.

I opened [new PR](https://github.com/flutter-tizen/engine/pull/216) to fix this issue.  
@JRazek Could you check it using this PR?

**Comment by bbrto21** [comment]
> I opened [new PR](https://github.com/flutter-tizen/engine/pull/216) to fix this issue. @JRazek Could you check it using this PR?

There's still a blank screen between the splash image and the first frame :(
